src/SignalR/common/Http.Connections/ref/Microsoft.AspNetCore.Http.Connections.netcoreapp.cs

```diff
@@ -30,7 +30,7 @@ public partial class ConnectionOptions
     }
     public partial class ConnectionOptionsSetup : Microsoft.Extensions.Options.IConfigureOptions<Microsoft.AspNetCore.Http.Connections.ConnectionOptions>
     {
-        public static System.TimeSpan DefaultDisconectTimeout;
+        public static System.TimeSpan DefaultDisconnectTimeout;
```

Comment:
Well that's unfortunate...

We can't really change that arbitrarily
cc @anurse

Reply:
Yeah, unfortunately it's a breaking change. That doesn't mean we can't take it though :). Let's raise this in an API review. I'll set things up for that.

Reply:
Currently it isn't `readonly`.